RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.12.0
+------
+* Add rich text styling to video captions
+* Prevent keyboard dismissal when switching between caption and text block on Android
+* Blocks that would be replaced are now hidden when add block bottom sheet displays
+* Tapping on empty editor area now always inserts new block at end of post
+
```

Comment:
Not sure if we are missing Media&Text and Group blocks from this list, or we want to hide them in 1.12.0 release?

Reply:
I confirmed with @etoledom that we do not want either of those blocks to be available in this release, [and they are guarded by a dev flag](https://github.com/WordPress/gutenberg/blob/rnmobile%2Frelease-1.12.0/packages%2Fblock-library%2Fsrc%2Findex.native.js#L146). 

I also made release builds of both iOS and Android and confirmed that those blocks are not selectable from the bottom modal block picker and if a post is shown on mobile that contains those blocks (created on the web), then the app just displays a block placeholder. Sounds like you were observing something different though, would you like to double-check this before we merge @marecar3 ?

Reply:
Thanks @mchowning for confirmation.
It's always good to check it :)

Reply:
>It's always good to check it :)

Absolutely! Thanks for bringing it up!